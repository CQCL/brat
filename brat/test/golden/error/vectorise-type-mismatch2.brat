bad :: Vec(Float, 1)
bad = 1 of 42
