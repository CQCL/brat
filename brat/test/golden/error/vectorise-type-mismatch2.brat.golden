Error in test/golden/error/vectorise-type-mismatch2.brat@FC {start = Pos {line = 2, col = 12}, end = Pos {line = 2, col = 14}}:
bad = 1 of 42
           ^^

  Type error: Expected something of type `Float` but got `42`

